test/unit/exception_handling_test.rb

```diff
@@ -194,6 +194,16 @@ class SmtpClientErrbackStub < SmtpClientStub
         assert_equal :skipped, @honeybadger_status
       end
 
+      should "include logging context in the exception data" do
+        ExceptionHandling.post_log_error_hook = method(:log_error_callback_config)
+        ExceptionHandling.log_error(StandardError.new("Some BS"), "mooo", nil, treat_like_warning: true, log_context_test: "contextual_logging")
```

Comment:
Please replace "Some BS".

Pointing this out because open source projects will often have Code of Conduct rules regarding appropriate language that are expected to be followed. We don't have one in this repo but we should still be on top of it.

Reply:
Changed